draft-huitema-dprive-dnsoquic.md

```diff
@@ -317,24 +294,21 @@ Therefore, a single client initiated DNS transaction consumes a single stream.
 This means that the client's first query occurs on QUIC stream 0, the second on 4,
 and so on.
 
-As specified in section 7 of "DNS Transport over TCP - Implementation
-Requirements" {{!RFC7766}}, resolvers are RECOMMENDED to
-support the preparing of responses in parallel and sending them out
-of order. In DoQ, they do that by sending responses on their specific
-stream as soon as possible, without waiting for availability of responses
-for previously opened streams.
-
 ### Transaction Errors
 
 Peers normally complete transactions by sending a DNS response on the
-transaction's stream, including in cases where the DNS response indicates an
-error. There are two exceptions: internal errors, and cancelled transactions.
+transaction's stream, including cases where the DNS response indicates a
+DNS error. For example, a Server
+Failure (SERVFAIL, {{!RFC1035}}) SHOULD be notified to the initiator of
+the transaction. 
+
```

Comment:
Maybe we need to be a bit more specific than "should be notified". How about "For example, a Server
Failure (SERVFAIL, {{!RFC1035}}) SHOULD be notified to the initiator of
the transaction by sending a DNS Response with Reply Code SERVFAIL on the transaction's stream."